From d682f37eeb0fec3de59f658d3b332d22fdcbf17b Mon Sep 17 00:00:00 2001
From: Junio C Hamano <gitster@pobox.com>
Date: Fri, 9 Oct 2009 13:44:53 -0700
Subject: [PATCH 10/22] s/mangement/management/

---

Message-ID: <7vskdss3ei.fsf@alter.siamese.dyndns.org>

> +Session State
> +-------------
> ...
> +retained and managed by the client process.  This permits simple
> +round-robin load-balancing on the server side, without needing to
> +worry about state mangement.

s/mangement/management/;
---
 Documentation/technical/http-protocol.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Documentation/technical/http-protocol.txt b/Documentation/technical/http-protocol.txt
index e7c05d2..7fe21a4 100644
--- a/Documentation/technical/http-protocol.txt
+++ b/Documentation/technical/http-protocol.txt
@@ -69,7 +69,7 @@ The Git over HTTP protocol (much like HTTP itself) is stateless
 from the perspective of the HTTP server side.  All state MUST be
 retained and managed by the client process.  This permits simple
 round-robin load-balancing on the server side, without needing to
-worry about state mangement.
+worry about state management.
 
 Clients MUST NOT require state management on the server side in
 order to function correctly.
-- 
1.8.4.rc4.527.g303b16c

